### DISCUSSION TITLE
Allow gists and repos to be in the same list.

### DISCUSSION THREAD

**Discussion by iiYese** [community/community#14503] [Other Feature Feedback, Questions, & Ideas]
Title.

**Comment by dfyockey** [comment]
Agreed. I just spent too much time finding that starred gists are listed with "Your gists" rather than "Your stars," which doesn't make sense because starred gists aren't your gists.

**Reply by jastigall392** [reply]
Helo

**Reply by turrap23** [reply]
Nb

**Reply by Hamaster2** [reply]
Mino

**Comment by ShadowCoder02** [comment]
I completely agree with you! It’s definitely confusing to have starred gists mixed in with "Your gists." Starred items typically belong in a dedicated "Your stars" section, which is intuitive since they're not actually your own content but something you're bookmarking or appreciating from others. Having a clearer distinction would save time and make navigating through gists more efficient. Hopefully, it’s something they’ll consider improving for better user experience!

**Comment by barbaryyazd09133545880** [comment]
باربری نیسان یزد 09133545880
www.barbariyazd.ir
